metrics/prometheus_test.go

```diff
@@ -44,6 +44,21 @@ func TestPrometheusCollector(t *testing.T) {
 	testPrometheusCollector(t, reg, "testdata/prometheus_metrics")
 }
 
+func TestPrometheusCollectorWithPerfAggregated(t *testing.T) {
```

Comment:
@dashpole Unit tests are available here. Do you think that I should add something?